### PR TITLE
fix bugs in ScorePosition #35

### PR DESCRIPTION
**Pull request by Toine-ddt**

### CODE REVIEW

File: src/ControlSurface/Controls/Position/ScorePosition.qml [hunk 1]
```diff
@@ -109,30 +116,24 @@ Rectangle {
...
-                // horizontal.y = tmpY * ( positionBackground / tmpDomain )
                 positionListModel.insert(positionListModel.count, {
                                              "_custom": s.Custom,
                                              "_id": s.id,
                                              "_uuid": s.uuid,
                                              "_controlX": tmpX,
                                              "_controlY": tmpY,
-                                             "_controlDomain": tmpDomain
+                                             "_controlDomain": tmpDomain,
+                                             "_controlMin": tmpMin
```
Comment (jcelerier) [inline src/ControlSurface/Controls/Position/ScorePosition.qml:136]: why no max in there ?

File: src/ControlSurface/Controls/Position/ScorePositionPoint.qml [hunk 2]
```diff
@@ -15,17 +15,18 @@ Rectangle {
     property string controlUuid
     property string controlSurfacePath
 
-    property real controlX: vertical.x * (controlDomain / position.height)
-    property real controlY: horizontal.y * (controlDomain / position.height)
+    property real controlX: controlMin + vertical.x * (controlDomain / position.height)
+    property real controlY: controlMin + horizontal.y * (controlDomain / position.height)
```
Comment (jcelerier) [inline src/ControlSurface/Controls/Position/ScorePositionPoint.qml:19]: same, in general when there's a min there's also a max ? but maybe there's a good reason

**Review by jcelerier — APPROVED** [review]

File: src/ControlSurface/Controls/Position/ScorePosition.qml (hunk 1 above)
Comment (Toine-ddt) [reply src/ControlSurface/Controls/Position/ScorePosition.qml:136]: because max is included in controlDomain = max-min

Comment (jcelerier) [reply src/ControlSurface/Controls/Position/ScorePosition.qml:136]: oh, then we'd say range instead of domain. But not dramatic

**Pull request merged**